parsers/powershell.c

```diff
@@ -801,5 +801,9 @@ extern parserDefinition* PowerShellParser (void)
 	def->parser     = findPowerShellTags;
 	def->keywordTable = PowerShellKeywordTable;
 	def->keywordCount = ARRAY_SIZE (PowerShellKeywordTable);
+
+	def->versionCurrent = 1;
+	def->versionAge = 1;
+
```

Comment:
This change should be part of the first commit.

Reply:
Done